api/src/main/java/io/strimzi/api/kafka/model/bridge/KafkaBridgeAdminClientSpec.java

```diff
@@ -29,6 +31,6 @@ public class KafkaBridgeAdminClientSpec extends KafkaBridgeClientSpec {
     @Override
     @Description("The Kafka AdminClient configuration used for AdminClient instances created by the bridge.")
     public Map<String, Object> getConfig() {
-        return config;
+        return this.config != null ? this.config : emptyMap();
```

Comment:
Map.of()? Also, why does this need to do this in the first place?

Reply:
> Map.of()?

Why Map.of is better than emptyMap?

> why does this need to do this in the first place?

I'm using the lazy initialization as for additionalProperties.

Reply:
* Map.of() is what we try to use in all places unless we need a writable map.
* Isn't it better to simply use null instead of initializing it here?

Reply:
We were always initializing with an empty map, regardless if the get method was called or not, so we are actually reducing the number of initializations.

Reply:
> Isn't it better to simply use null instead of initializing it here?

I think that `null` would cause some NPE. For example here https://github.com/strimzi/strimzi-kafka-operator/blob/main/cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java#L412 if the configuration for the admin client is not set.

Reply:
Ok, then I guess we can stick with what we have here right now and assuming it can be immutable just use `Map.of`.